app/uk/gov/hmrc/mobilehelptosavefrontend/config/ErrorHandler.scala

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
why removed this?

Reply:
From play version 9 onwards uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler supports asynchronous rendering. And ErrorHandler.scala was extending FrontendErrorHandler, that is why I removed it.